portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,33 +30,34 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.FetchOptions.Builder;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   private int numComments = 10;
+  private int page = 0;
```

Comment:
Preferably the page number (and maybe even the numComments) would be remembered by the browser so that if multiple users are using the page their values won't conflict. If users could sign in, then it might make sense to store them server side but separate for each user. I think for now this is ok, though.

Reply:
I could store numComments and page in the sessionStorage with JS, but how could I get them to the server?